packages/block/src/header.ts

```diff
@@ -208,6 +218,12 @@ export class BlockHeader {
       this._common.setHardforkByBlockNumber(number.toNumber())
     }
 
+    if (this._common.isActivatedEIP(1559) && baseFeePerGas === undefined) {
+      this.baseFeePerGas = new BN(1)
```

Comment:
is this default `1` value defined in the spec somewhere? can't seem to locate it. just wanted to verify it would be a proper default value. if undefined do we maybe want to set it to `INITIAL_BASE_FEE` instead? (in the spec as `1000000000`)